### ISSUE TITLE
/api/v2/search ignores offset when searching for a status url

### ISSUE THREAD

**Issue by connyduck** [tootsuite/mastodon#13083]
### Expected behaviour

`GET https://mastodon.social/api/v2/search?q=https:%2F%2Fmastodon.social%2F@ConnyDuck%2F101501446522432586&resolve=true&offset=1` returns an empty result
### Actual behaviour

`GET https://mastodon.social/api/v2/search?q=https:%2F%2Fmastodon.social%2F@ConnyDuck%2F101501446522432586&resolve=true&offset=1` returns the status expected at offset 0. Looks like the offset does not matter, there will always be one status returned

### Specifications

Problem occurs on all Mastodon instances I have access to, e.g. Mastodon.social which currently runs v3.1.0

Basically, this is the same bug as https://github.com/tootsuite/mastodon/issues/11365 but with statuses instead of accounts

**Comment by Gargron** [comment]
I'm confused by the expectation. A search request to resolve a specific URL can only return a single result. In what scenario would you want pagination params to have any meaning there?

**Comment by Gargron** [comment]
If `resolve` is false, then the search query *will* try to find available statuses that *include* said URL as text, and in that scenario pagination does matter, but I don't think that code path is used at all when `resolve` is true.

**Comment by connyduck** [comment]
If you are not going to fix it, then please explain this weird behavior in the api docs.
